extensions/db/src/main/resources/META-INF/mule-module.properties.old

```diff
@@ -0,0 +1,82 @@
+module.name=db
+
+artifact.export.classPackages=org.mule.extension.db.config,\
```

Comment:
this class should actually be deleted. It's a left over from the pre isolation era